File: wear/src/main/java/com/example/de/testssapplication/MainActivity.java
package com.example.de.testssapplication;

import android.content.Intent;
import android.os.Bundle;
import android.support.wearable.activity.WearableActivity;
import android.view.View;
import android.widget.Button;

public class MainActivity extends WearableActivity {
    //Log-Tag
    public static String TAG = "Wear-MainActivity";

    private Button snButton;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        snButton  = (Button) findViewById(R.id.sn_button);
        snButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(getApplicationContext(), DirectionNotifier.class);
                intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP | Intent.FLAG_ACTIVITY_NEW_TASK);
                startActivity(intent);
            }
        });

    }
}

File: wear/src/main/java/com/example/de/testssapplication/MainActivity.kt
package com.example.de.testssapplication

import android.content.Intent
import android.os.Bundle
import android.support.wearable.activity.WearableActivity
import android.view.View
import android.widget.Button

class MainActivity : WearableActivity() {

    private var snButton: Button? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        snButton = findViewById<View>(R.id.sn_button) as Button
        snButton!!.setOnClickListener {
            val intent = Intent(applicationContext, DirectionNotifier::class.java)
            intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP or Intent.FLAG_ACTIVITY_NEW_TASK)
            startActivity(intent)
        }

    }

    companion object {
        //Log-Tag
        var TAG = "Wear-MainActivity"
    }
}
